### PR TITLE
refactored variables

### PR DESCRIPTION
**Pull request by dshcheung**

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb [hunk 1]
```diff
@@ -128,18 +111,18 @@ app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', 'UserServ
     }
   };
 
-  $scope.searchWord = "Ingredient";
+  $scope.searchWord == "Ingredient"
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb]: What's this condition for?

File: app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb [hunk 2]
```diff
@@ -159,19 +141,15 @@ app.controller('RecipeFinderCtrl', ['$scope', '$http', '$routeParams', 'UserServ
     $scope.search_parameters.ingredients.splice(index,1);
     $scope.submitParams();
   }
-  // meals search
-  // toggle selection for a given food by name
-  $scope.toggleMeals = function toggleMeals(meal) {
-    var idx = $scope.search_parameters.categories.indexOf(meal);
 
+  // toggle selection for a given food by name
+  $scope.toggleCategory = function (category) {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb:146]: Good refactoring

**Comment by h8rry** [comment]
The rest looks good

File: app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb (hunk 1 above)
Comment (dshcheung) [inline app/assets/javascripts/angular/controllers/pages/recipe_finder_controller.js.erb]: i think i was angry and hit my keyboard :P

**Pull request merged**